docs/_templates/class.rst

```diff
@@ -1,20 +1,61 @@
+.. |img| replace:: ``img`` (:obj:`numpy.array`): a numpy array (height, width, channels)
+   representation of the image, in BGR format
+
+.. |bboxes| replace:: ``bboxes`` (:obj:`numpy.array`): a numpy array (N,4)
+   containing bounding box information of detected objects. N corresponds to the number of objects detected,
+   and each bounding box is represented as (x1,y1,x2,y2),
+   (x,y) coordinates of the top-left corner and bottom right corner of the bounding box respectively.
+
+.. |bbox_labels| replace:: ``bbox_labels`` (:obj:`numpy.array`): a numpy array (N) strings,
+   representing the labels of detected objects. The order corresponds to `bboxes` and `bbox_scores`.
+
+.. |bbox_scores| replace:: ``bbox_scores`` (:obj:`numpy.array`): a numpy array (N). [0,1]
+   of the confidence scores for detected objects. The order corresponds to `bboxes` and `bbox_labels`.
+
+.. |keypoints| replace:: ``keypoints`` (:obj:`numpy.array`): a numpy array (N, keypoints, 2) containing the
+   xy coordinates for detected poses. N represents the number of detected poses, and K represents indiviudal
+   keypoints. For keypoints that have low confidence score (below threshold), it will be placed by `-1`.
+
+.. |keypoint_scores| replace:: ``keypoints_scores`` (:obj:`numpy.array`): a numpy array (N, keypoints, 1) containing
+   the confidence scores of detected pose. N represents the number of detected poses, and K represents indiviudal
+   keypoints.
+
+.. |keypoints_conns| replace:: ``keypoints_conns`` (:obj:`numpy.array`): a numpy array containing the coordinates
+   of keypoint connections between adjacent keypoints pairs, if both keypoints are detected.
+
+.. |pipeline_end| replace:: ``pipeline_end`` (:obj:`bool`): A boolean that evaluates as ``True`` when
+   the pipeline is completed. Suitable for operations that require the complete inference pipeline to
+   be completed before running.
+
+.. |filename| replace:: ``filename`` (:obj:`str`): the filename of video/image being read.
+
+.. |fps| replace :: ``fps`` (:obj:`float`): the FPS of recorded video, upon filming.
```

Comment:
It is now `saved_video_fps` following Zi Zhao's PR today.

Reply:
updated